src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -828,11 +839,47 @@ public void SarifTransformerTests_ToCurrent_NotificationExceptionWithStack()
                   },
                   ""frames"": [
                     {
+                      ""location"": {
+                        ""physicalLocation"": {
+                          ""fileLocation"": {
+                            ""uri"": ""file:///C:/src/main.cs""
+                          },
+                          ""region"": {
+                            ""startLine"": 15,
+                            ""startColumn"": 9
+                          }
+                        },
+                        ""fullyQualifiedLogicalName"": ""Rules.SecureHashAlgorithmRule.Evaluate-0"",
```

Comment:
How does the transformer know that this isn't the same `Rules.SecureHashAlgorithm.Evaluate` that's already in the dictionary? That is, why did it decide to disambiguate. (Don't answer yet; this is a note-to-self. I'll keep reading.)

Reply:
@EasyRhin0 It looks like we still have the bug. It shouldn't be necessary to disambiguate `Rules.SecureHashAlgorithmRule.Evaluate` -- because the v1 `logicalLocation` with that key contained only `"name": "Evaluate"`, which is correct, and `"kind": "some kind"`, which should not participate in the comparison.